package.json

```diff
@@ -29,7 +29,7 @@
     "fix:js": "npm run lint:js -- --fix",
     "fix:prettier": "npm run lint:prettier -- --write",
     "fix": "npm-run-all -l fix:js fix:prettier",
-    "test:only": "cross-env NODE_ENV=test jest",
+    "test:only": "cross-env NODE_ENV=test NODE_OPTIONS=--experimental-vm-modules jest",
```

Comment:
Tests were failing without this.

<img width="797" alt="Screenshot 2024-01-16 at 10 30 46 AM" src="https://github.com/webpack-contrib/expose-loader/assets/46647141/44673931-48bc-4f1b-b7ea-62d72dbf8615">